src/bo4e/bo/buendelvertrag.py

```diff
@@ -27,5 +34,35 @@ class Buendelvertrag(Geschaeftsobjekt):
 
     # required attributes
     bo_typ: BoTyp = BoTyp.BUENDELVERTRAG
+
+    # pylint: disable=duplicate-code
+    #: Eine im Verwendungskontext eindeutige Nummer für den Vertrag
+    vertragsnummer: str
+    #: Hier ist festgelegt, um welche Art von Vertrag es sich handelt. Z.B. Netznutzungvertrag
+    vertragsart: Vertragsart
```

Comment:
ich glaube du hast konsequent die docstrings an die falsche stelle gesetzt.
sollten die nicht nach/unter dem attribute sein statt davor/darüber?

Ich verstehe gerade selbst nicht, wie genau es funktioniert:
```python
        sparte_docstring = inspect.getdoc(buendelvertrag.sparte)
        assert "Sparte" in sparte_docstring # <-- klappt ☺
        vertragsnummer_docstring = inspect.getdoc(buendelvertrag.vertragsnummer)
        assert "Nummer" in vertragsnummer_docstring # <-- klappt nicht. 🤷‍♀️
```

Reply:
Keine Ahnung, warum dein assert nicht funktioniert, um ehrlich zu sein. In der Doku ist es auf jeden Fall richtig.